### ISSUE TITLE
UiCollapsible with UiSelect

### ISSUE THREAD

**Issue by EmilMoe** [JosephusPaye/Keen-UI#355]
### Problem

When having an UiSelect inside UiCollapsible the overflow is prevented. However adding this makes the UiSelect works again:

~~~~
    .ui-collapsible {
        .ui-collapsible__body-wrapper {
            overflow: initial;
        }
    }
~~~~

But then when you close it, the content is visible until it has been closed.

### Possible solution

Would it be possible that by default it's `overflow: initial;` and when you close it, and until it's closed, it's changed to `overflow: auto;`?

**Comment by JosephusPaye** [comment]
Hmm, that might fix it. I'll look into it, thanks.

**Comment by NovikovRoman** [comment]
temporary (constant? :smile:) solution

style.css
`
…
.ui-collapsible__body-wrapper {
    overflow: initial;
}

.ui-collapsible__body-wrapper.ui-collapsible--transition-toggle-enter-active,
.ui-collapsible__body-wrapper.ui-collapsible--transition-toggle-leave-active {
    overflow: hidden;
}
…
`

**Comment by JosephusPaye** [comment]
This is fixed in `next` with the collapsible [rewrite](https://github.com/JosephusPaye/Keen-UI/commit/435677b1bb8f3fef5c78cf048d05f42b6286823a).

<img src="https://user-images.githubusercontent.com/5924865/51816338-3e214380-231a-11e9-9645-ca65195caf52.png" width="350px">

**Comment by JosephusPaye** [comment]
Fix released in [v1.1.0](https://github.com/JosephusPaye/Keen-UI/releases/tag/v1.1.0).
